nuxeo-features/nuxeo-automation/nuxeo-automation-client/src/main/java/org/nuxeo/ecm/automation/client/jaxrs/spi/marshallers/DocumentsMarshaller.java

```diff
@@ -92,6 +92,8 @@ protected Documents readPaginableDocuments(JsonParser jp) throws IOException {
                 docs.setCurrentPageIndex(jp.getIntValue());
             } else if ("entries".equals(key)) {
                 readDocumentEntries(jp, docs);
+            } else if ("aggregations".equals(key)) {
```

Comment:
This part doesn't seem tested in this PR. This is automation-client so I can understand if it's too complex.
Can this me "aggregates" instead of "aggregations" for consistency?

Reply:
TBH this is just so I can check whether the `skipAggregates` flag is working since our automation client returns pageable documents which did not include any aggregates information and I couldn't get it to return a "raw" JsonNode like I did with the HttpAutomationClient in master.

Reply:
Can't change it to "aggregates" as it would break the REST API. This has been called "aggregations" since the beginning 🤷‍♂️